test/functional/test_site/expected/sub_site/testReuse._include_.html

```diff
@@ -18,4 +18,9 @@
   <a href="https://dummyimage.com/600x400/000/fff">External Image</a>
   <a href="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">Link to picture</a>
   <a id="namedAnchor">Named Anchor</a>
-  <a>Anchor with no attributes</a></p>
\ No newline at end of file
+  <a>Anchor with no attributes</a></p>
+<p>Within DIV tag:</p>
+<div id="imageTest">
+  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <pic src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png"></pic>
+</div>
```

Comment:
I think a newline should be placed at the end of this file 🙂

Might be useful to see if your IDE has the option to automatically add new lines at the end of the file when necessary!

Reply:
Thanks for the review!
Actually this file is the auto generated expected file, so having a newline will cause the test to fail...

Reply:
Ah right, that makes sense! Thanks for pointing it out 🙂

Reply:
We should add it during generation then. 🙊